moveit_ros/moveit_servo/include/moveit_servo/servo_calcs.h

```diff
@@ -49,8 +49,6 @@
 #include <geometry_msgs/msg/twist_stamped.hpp>
 #include <moveit/planning_scene_monitor/planning_scene_monitor.h>
 #include <moveit/robot_model_loader/robot_model_loader.h>
-#include <moveit_msgs/srv/change_drift_dimensions.hpp>
-#include <moveit_msgs/srv/change_control_dimensions.hpp>
```

Comment:
Please open a PR to moveit_msgs to remove these types.

Reply:
Sure , Will do